cypress/e2e/pages/articles/tests.js

```diff
@@ -61,179 +44,80 @@ export const testsThatFollowSmokeTestConfig = ({
-              cy.request({
-                url: href,
-                failOnStatusCode: false,
-              }).then(resp => {
-                expect(resp.status).to.not.equal(404);
-              });
-            });
-        });
-      }
-
-      if (serviceHasTimestamp(service)) {
-        it('should render a timestamp', () => {
-          cy.get('time')
-            .eq(0)
-            .should('exist')
             .should('be.visible')
-            .should('have.attr', 'datetime')
-            .should('not.be.empty');
+            .should('to.have.descendants', 'img')
+            .should('to.have.descendants', 'figcaption')
+            .within(() => cy.get('noscript').should('not.exist'));
         });
       }
-
-      describe('Media Player', () => {
-        it('should have a visible caption beneath a mediaplayer', () => {
-          const media = getBlockData('video', articlesData);
-          if (media) {
-            const captionBlock = getBlockByType(media.model.blocks, 'caption');
-
-            if (captionBlock) {
-              const { text } =
-                captionBlock.model.blocks[0].model.blocks[0].model;
-
-              cy.get('figcaption')
-                .eq(1)
-                .within(() => {
-                  cy.get('[data-testid="caption-paragraph"]')
-                    .eq(0)
-                    .should('be.visible')
-                    .should('contain', text);
-                });
-            }
-          }
-        });
+    }
+
+    if (serviceHasInlineLink(service) && Cypress.env('APP_ENV') !== 'live') {
+      it('should have an inline link to an article page', () => {
+        cy.get('a[href*="/articles/"]')
+          .should('have.attr', 'href')
+          .then(href => {
+            cy.request({
+              url: href,
+              failOnStatusCode: false,
+            }).then(resp => {
+              expect(resp.status).to.not.equal(404);
+            });
+          });
       });
-
-      describe('Social Embeds', () => {
-        const availableSocialMediaOnPage = [];
-        const socialIsOnPage = social =>
-          availableSocialMediaOnPage.includes(social);
-        before(() => {
-          availableSocialMediaOnPage.push(
-            ...getAllBlocksDataByType('social', articlesData).map(
-              block => block.model.providerName,
-            ),
-          );
+    }
+
+    if (serviceHasTimestamp(service)) {
+      it('should render a timestamp', () => {
+        cy.get('time')
+          .eq(0)
+          .should('exist')
+          .should('be.visible')
+          .should('have.attr', 'datetime')
+          .should('not.be.empty');
+      });
+    }
+
+    // eslint-disable-next-line no-only-tests/no-only-tests
+    describe('Social Embeds', () => {
+      let articleId;
+      before(() => {
+        cy.url().then(url => {
+          // eslint-disable-next-line prefer-destructuring
+          articleId = url.match(/\/([^/]+?)(?:\.[^/.]+)?$/)[1];
```

Comment:
Can we destructure this as suggested by linter? 
```suggestion
          const [, articleId] = url.match(/\/([^/]+?)(?:\.[^/.]+)?$/);
```